src/shared/debug_op.c

```diff
@@ -338,6 +340,65 @@ void _merror(const char * file, int line, const char * func, const char *msg, ..
     va_end(args);
 }
 
+static char* _smsg(int level, const char * file, int line, const char * func, const char *msg, va_list args){
+    
+    char* str = NULL;
+    os_malloc(OS_BUFFER_SIZE, str);
+    str[0] = '\0';
+    
+    const char *strlevel[5]={
+      "DEBUG",
+      "INFO",
+      "WARNING",
+      "ERROR",
+      "CRITICAL",
+    };
+
+    if (isDebug())
```

Comment:
According to the [code style guide](https://github.com/wazuh/wazuh/wiki/Coding-style-guide-%28C%29#conditionals):
```c
    if (isDebug()) {
        (void) snprintf(str, OS_BUFFER_SIZE, "%s:%d at %s(): %s: ", file, line, func, strlevel[level]);
    } else {
        (void) snprintf(str, OS_BUFFER_SIZE, "%s: ", strlevel[level]);
    }
```